src/app/navbar/navbar.component.scss

```diff
@@ -1,48 +1,64 @@
-nav.navbar {
+:host {
+  --ds-collapsible-navbar-height: auto;
+
+  nav.navbar {
     background-color: var(--ds-navbar-bg);
     align-items: baseline;
-}
+    padding: 0;
+  }
 
-/** Mobile menu styling **/
-@media screen and (max-width: map-get($grid-breakpoints, md)-0.02) {
+  /** Mobile menu styling **/
+  @media screen and (max-width: map-get($grid-breakpoints, md)-0.02) {
     .navbar {
-        width: 100%;
-        background-color: var(--bs-white);
-        position: absolute;
-        overflow: hidden;
-        height: 0;
-        z-index: var(--ds-nav-z-index);
-        &.open {
-          height: auto;
-          min-height: 100vh; //doesn't matter because wrapper is sticky
-          border-bottom: 1px var(--ds-header-navbar-border-bottom-color) solid; // open navbar covers header-navbar-wrapper border
-        }
+      width: 100%;
+      background-color: var(--bs-white);
+      position: absolute;
+      overflow: hidden;
+      height: 0;
+      z-index: var(--ds-nav-z-index);
+      &.open {
+        height: auto;
+        min-height: 100vh; //doesn't matter because wrapper is sticky
+        border-bottom: 1px var(--ds-header-navbar-border-bottom-color) solid; // open navbar covers header-navbar-wrapper border
+      }
     }
-}
+  }
 
-@media screen and (min-width: map-get($grid-breakpoints, md)) {
+  @media screen and (min-width: map-get($grid-breakpoints, md)) {
     .reset-padding-md {
-        margin-left: calc(var(--bs-spacer) / -2);
-        margin-right: calc(var(--bs-spacer) / -2);
+      margin-left: calc(var(--bs-spacer) / -1);
+      margin-right: calc(var(--bs-spacer) / -1);
     }
-}
+  }
 
-/* TODO remove when https://github.com/twbs/bootstrap/issues/24726 is fixed */
-.navbar-expand-md.navbar-container {
+  /* TODO remove when https://github.com/twbs/bootstrap/issues/24726 is fixed */
+  .navbar-expand-md.navbar-container {
     @media screen and (max-width: map-get($grid-breakpoints, md)-0.02) {
-        > .navbar-inner-container {
-            padding: 0 var(--bs-spacer);
-        }
-        padding: 0;
+      > .navbar-inner-container {
+        padding: 0 var(--bs-spacer);
+      }
+      padding: 0;
+    }
+  }
+
+  #main-navbar ::ng-deep {
+    .ds-menu-item, .ds-menu-toggler-wrapper {
+      white-space: nowrap;
+      text-decoration: none;
+    }
+
+    .dropdown-menu {
+      padding: 0.5rem !important;
     }
-}
 
-.navbar-nav ::ng-deep {
-  a.nav-link, .btn.nav-link {
-    color: var(--ds-navbar-link-color);
+    .ds-menu-item {
+      display: block;
+      color: var(--ds-navbar-link-color);
+      padding: 0.5rem 0;
```

Comment:
This should be (derived from) a variable

Reply:
It looks like @artlowel 's feedback on this line wasn't resolved or answered.  I think he was asking for the `padding` to be derived from a variable

Reply:
@tdonohue @artlowel I'm sorry, I though I already replied to this but I cannot see my answer anymore. In this case I chose not to use a variable because it would affect only the _base_ theme, and not the _dspace_ theme, and in my opinion having a global variable that does not change anything on the default _dspace_ theme may be confusing